src/web/core/src/main/java/org/geoserver/web/wicket/GeoServerTablePanel.html

```diff
@@ -12,6 +12,7 @@
 		<div class="float-end">
 			<input id="filter" wicket:id="filter" class="me-1 search" type="text" size="35" title="Search" placeholder="Search" wicket:message="placeholder:Search"/>
 			<input wicket:id="submit" class="d-none" type="submit" value="Search"/>
+			<span class="help-link" title="Enclose the search term in quotes for exact-term search."></span>
```

Comment:
This should be subject to internationalization (you can provide support for english only for starters, in ``GeoServerApplication.properties``). Also, make sure it alignes with actual implementation (single or double quotes, see above).

Reply:
Added. Thanks.